shell/platform/darwin/ios/framework/Source/FlutterDartProject.mm

```diff
@@ -207,7 +207,11 @@ + (NSString*)flutterAssetsName:(NSBundle*)bundle {
 }
 
 + (NSString*)lookupKeyForAsset:(NSString*)asset {
-  return [self lookupKeyForAsset fromBundle:[NSBundle mainBundle]];
+  NSBundle* bundle = [NSBundle bundleWithIdnetifier:[FlutterDartProject defaultBundleIdentifier]];
```

Comment:
FYI on this: it is mainly for allowing the documented way (https://flutter.dev/docs/development/ui/assets-and-images#ios) to work with the default setting of Flutter iOS app, which has default bundle id but may have different bundle name.